modules/anki/hm.nix

```
@@ -0,0 +1,347 @@
+{
+  config,
+  lib,
+  mkTarget,
+  pkgs,
+  ...
+}:
+mkTarget {
+  name = "anki";
+  humanName = "Anki";
+  autoEnable = config.programs.anki.enable;
+  autoEnableExpr = "programs.anki.enable";
+
+  configElements = lib.singleton (
```

We already call `lib.singleton` internally:

```suggestion
  configElements =
```